common/bin/oo-diagnostics

```diff
@@ -1204,6 +1204,29 @@ class OODiag
 
   end
 
+  def test_auth_keys_salt_failure
+    if Rails.env.development?
+      msg_log = `grep \'401 Unauthorized\' /var/log/openshift/broker/development.log`
+    else
+      msg_log = `grep \'401 Unauthorized\' /var/log/openshift/broker/production.log`
+    end
```

Comment:
Would be better to use a variable for the log filename and interpolate that variable into the command.